hg-fast-export.sh

```diff
@@ -187,7 +183,7 @@ $(
       --mapping "$GIT_DIR/$PFX-$SFX_MAPPING" \
       --heads "$GIT_DIR/$PFX-$SFX_HEADS" \
       --status "$GIT_DIR/$PFX-$SFX_STATE" \
-      "$@" 3>&- || _e1=$?
+      s"${HG2GIT_ARGS[@]}" 3>&- || _e1=$?
```

Comment:
Does the  `s"` have a special meaning in sh or is something that got there by mistake? I think the latter as I can't get it to do something special on the command line.

Reply:
Oops, merely a typo, sorry.